lib/travis/api/v3/services/crons/start.rb

```diff
@@ -1,9 +1,29 @@
 module Travis::API::V3
   class Services::Crons::Start < Service
-
     def run!
-      query.start_all()
+      Travis.logger.info "DB connection to: #{ActiveRecord::Base.connection.current_database}"
+      loop do
```

Comment:
@svenfuchs: I was using run_periodically from [here](https://github.com/travis-ci/travis-support/blob/master/lib/core_ext/kernel/run_periodically.rb) before, but `Thread.new` was crashing the whole script. Haven't figured out why yet. But this loop runs fine